specs/ClientCertificate.md

```diff
@@ -559,14 +559,18 @@ namespace Microsoft. Web. WebView2. Core
         CoreWebView2ClientCertificateKind Kind { get; };
 
         String ToPemEncoding();
-        /// Converts this to a System.Security.Cryptography.X509Certificates.X509Certificate2.
-        /// This is only for the .NET API, not the WinRT API.
+        // Converts this to a System.Security.Cryptography.X509Certificates.X509Certificate2.
+        // This is only for the .NET API, not the WinRT API.
         X509Certificate2 ToX509Certificate2(CoreWebView2ClientCertificate coreWebView2ClientCertificate);
+
+        // Converts this to a Windows.Security.Cryptography.Certificates.Certificate.
```

Comment:
Just the note about this only existing for WinRT or .NET should be //, the other part about what the method does is good to have as ///
```suggestion
        /// Converts this to a Windows.Security.Cryptography.Certificates.Certificate.
```